app/helpers/application_html_formatters_helper.rb

```diff
@@ -74,7 +75,9 @@ def format_code_block(code, language = nil)
       end
     end
 
-    format_with_pipeline(DefaultCodePipeline, code)
+    formatted_code = DefaultCodePipeline.to_document(code)
```

Comment:
That's the way what I have tried at the very beginning, from what I read from their documentation and source code, they don't allow that. 

`HTML::Pipeline.new` expects actual class as the prams, and the classes are instantiates in the framework internally.